connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorMakerConfigTest.java

```diff
@@ -77,20 +79,23 @@ public void testReplicationConfigProperties() {
 
     @Test
     public void testClientConfigProperties() {
+        String clusterABBootstrap = "127.0.0.1:9092, 127.0.0.2:9092";
+        String clusterBBoostrap = "127.0.0.3:9092, 127.0.0.4:9092";
```

Comment:
`clusterBBoostrap` -> `clusterBBootstrap`

Reply:
fixed the typo